testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/SecurityCommandsTestCase.java

```diff
@@ -77,11 +97,271 @@
+                        + TestSuiteEnvironment.getServerAddress() + ":"
+                        + TestSuiteEnvironment.getServerPort()).
+                addCliArgument("--connect");
+
+        try {
+            cli.executeInteractive();
+            cli.clearOutput();
+
+            Path jbossStandaloneConfig = Paths.get(TestSuiteEnvironment.getJBossHome() +"/standalone/configuration");
+            Path KSFile = jbossStandaloneConfig.resolve(KEYSTORE_FILE);
+
+            //Copy the pre-generated key stores to the server config
+            Files.copy(AcmeMockServerBuilder.class.getResourceAsStream(ACCOUNTS_KEYSTORE_FILE_NAME),
+                    jbossStandaloneConfig.resolve(ACCOUNTS_KEYSTORE_FILE_NAME),
+                    java.nio.file.StandardCopyOption.REPLACE_EXISTING);
+            Files.copy(AcmeMockServerBuilder.class.getResourceAsStream(KEYSTORE_FILE), KSFile, java.nio.file.StandardCopyOption.REPLACE_EXISTING);
+            //Check that the key-store does not contain server certificate
+            assertFalse(loadKeyStore(KSFile.toString(), KEYSTORE_PASSWORD).containsAlias(certificateAlias));
+
+            Assert.assertTrue(cli.pushLineAndWaitForResults("security enable-ssl-management --interactive --no-reload"
+                            + " --lets-encrypt"
+                            + " --ca-account=FakeCaAccount", "Key-store file name (default management.keystore):" ));
+            Assert.assertTrue(cli.pushLineAndWaitForResults(KEYSTORE_FILE, "Password (blank generated)"));
+            Assert.assertTrue(cli.pushLineAndWaitForResults(KEYSTORE_PASSWORD, "Your domain name(s) (must be accessible by the Let's Encrypt server at 80 & 443 ports) [example.com,second.example.com]"));
+            Assert.assertTrue(cli.pushLineAndWaitForResults("inlneseppwkfwew.com", "Alias (blank generated):"));
+            Assert.assertTrue(cli.pushLineAndWaitForResults(certificateAlias, "Enable SSL Mutual Authentication"));
+            Assert.assertTrue(cli.pushLineAndWaitForResults("n", "Do you confirm"));
+            Assert.assertTrue(cli.pushLineAndWaitForResults("y", null));
+
+            //Check that the security command failed and the keystore does not contain the new certificate alias
+            assertFalse(loadKeyStore(KSFile.toString(), KEYSTORE_PASSWORD).containsAlias(certificateAlias));
+
+        } catch (Throwable ex) {
+            throw new Exception(cli.getOutput(), ex);
+        } finally {
+            cli.destroyProcess();
+        }
+    }
+
+    @Test
+    public void testEnableLetsEncryptSSLInteractiveConfirm() throws Exception {
+        testEnableLetsEncryptSSLInteractiveConfirm(null, false);
+    }
+
+    @Test
+    public void testEnableLetsEncryptSSLInteractiveConfirmUseCaAccount() throws Exception {
+        testEnableLetsEncryptSSLInteractiveConfirm(null, true);
+    }
+
+    private void testEnableLetsEncryptSSLInteractiveConfirm(String mgmtInterface, boolean useCaAccount) throws Exception {
```

Comment:
Why have this method parameter mgmtInterface? Both tests what it calls with null this parameter.

Reply:
Removed mgmtInterface.